ingest/build-configs/ncbi/bin/ncbi-virus-url

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+"""
+Generate URL to download all virus sequences and their curated metadata for a
+specified NCBI Taxon ID from GenBank via NCBI Virus.
+
+The URL this program builds is based on the URL for SARS-CoV-2 constructed with
+
+    https://github.com/nextstrain/ncov-ingest/blob/2a5f255329ee5bdf0cabc8b8827a700c92becbe4/bin/genbank-url
+
+and observing the network activity at
+
+    https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide
+"""
+from urllib.parse import urlencode
+from typing import List, Optional
+import argparse
+
+def parse_args():
```

Comment:
```suggestion
def parse_args() -> argparse.Namespace:
```

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/40/commits/eb5e068ef2d4260b01c7ac49fa805b539fb05e83